docs/src/development/environment-setup.md

```diff
@@ -29,6 +29,11 @@ $ cargo check
 > and use the newest [universal-sierra-compiler](https://github.com/software-mansion/universal-sierra-compiler)
 > after setting up the development environment, otherwise the tests will fail.
 
+> 📝 **Note**
+>
+> To run `sncast` tests, it is required to set `SEPOLIA_RPC_URL` environment variable, which is
+> used to fork the Sepolia testnet network in the devnet tests. This variable can be set in the `.env` file.
+
```

Comment:
I think notes should be used for additional info. Thise seems like core prerequisite, can se just write it as a standard text?

Reply:
Changed to a standard text. I think it would be beneficial to modify the environment setup docs. For instance running `install_denvet.sh` seems to be a core prerequisite as well, but it's mentioned in the note above.

Reply:
Yup, there is even a open PR for that #1983